draft-ietf-suit-manifest.md

```diff
@@ -1339,7 +1339,7 @@ and a page within this category for SUIT manifests.
 
 IANA is also requested to create several registries defined in the subsections below.
 
-For each registry, values 0-255 are Standards Action and 256 or greater are Specification Required. Negative values -255 to 0 are Standards Action, and -256 and lower are Private Use.
+For each registry, values 0-255 are Standards Action and 256 or greater are Specification Required with Expert Review. Negative values -255 to 0 are Standards Action, and -256 and lower are Private Use.
```

Comment:
This is redundant.  RFC 8126 says:

```
4.6.  Specification Required

   For the Specification Required policy, review and approval by a
   designated expert (see Section 5) is required, and the values and
   their meanings must be documented in a permanent and readily
   available public specification, in sufficient detail so that
   interoperability between

4.5.  Expert Review

   For the Expert Review policy, review and approval by a designated
   expert (see Section 5) is required.

```

So 4.6 always implies 4.5.

Reply:
thanks, resolved!